config/locations_pruned.yml

```diff
@@ -34,57 +29,22 @@ target_locations:
       - 71.2
     area: 20
     power: 615
-  5MW Solar Power Plant Varroc:
-    location:
-      - 21.059537
-      - 74.375055
-    power: 5
-  Welspun Solar MP project:
-    location:
-      - 24.6907
-      - 75.1347
-    area: 3
-    power: 151
-  Rewa Ultra Mega Solar:
-    location:
-      - 24.480278
-      - 81.574444
-    area: 6.4
-    power: 750
-  Bitta Solar Power Plant:
+  BOSCH Solar Power Plant:
     location:
-      - 23.262778
-      - 69.024167
-    area: 1.4
-    power: 40
+      - 15.981156
+      - 74.998076
+    area: 0.364
+    power: 115
   Indira Paryavaran Bhawan:
     location:
       - 28.585154
       - 77.218347
     area: 0.0046
     power: 0.93
-  Kurnool Ultra Mega Solar Park:
-    location:
-      - 15.681522
-      - 78.283749
-    area: 24
-    power: 1000
-  Chennai Metropolitan Area:
-    location:
-      - 13.083667
-      - 80.268667
   Adani Green Energy Tamilnadu Limited:
     location:
       - 9.332771
       - 78.388731
-  Solar power plant Koppal:
```

Comment:
Why you deleted any location from `locations_pruned.yml`? All of these were used in my paper: https://www.mdpi.com/2076-3417/12/17/8852